test/examples_test.go

```diff
@@ -54,60 +53,52 @@ func GetCreatedTektonCrd(input []byte, kind string) (string, error) {
 }
 
 func waitValidatePipelineRunDone(t *testing.T, c *clients, pipelineRunName string) {
-	err := WaitForPipelineRunState(c, pipelineRunName, pipelineRunTimeout, Succeed(pipelineRunName), pipelineRunName)
-
-	if err != nil {
+	if err := WaitForPipelineRunState(c, pipelineRunName, pipelineRunTimeout, Succeed(pipelineRunName), pipelineRunName); err != nil {
 		t.Fatalf("Failed waiting for pipeline run done: %v", err)
 	}
-	return
 }
 
 func waitValidateTaskRunDone(t *testing.T, c *clients, taskRunName string) {
 	// Per test basis
-	err := WaitForTaskRunState(c, taskRunName, Succeed(taskRunName), taskRunName)
-
-	if err != nil {
+	if err := WaitForTaskRunState(c, taskRunName, Succeed(taskRunName), taskRunName); err != nil {
 		t.Fatalf("Failed waiting for task run done: %v", err)
 	}
-	return
 }
 
-// SubstituteEnv substitutes docker repos and bucket paths from the system
-// environment for input to allow tests on local clusters. It also updates the
-// namespace for ServiceAccounts so that they work under test
-func SubstituteEnv(input []byte, namespace string) ([]byte, error) {
+// substituteEnv substitutes docker repos and bucket paths from the system
+// environment for input to allow tests on local clusters. It also unsets the
+// namespace for ServiceAccounts so that they work under test.
+func substituteEnv(input []byte, namespace string) ([]byte, error) {
+	// Replace the placeholder image repo with the value of the
+	// KO_DOCKER_REPO env var.
 	val, ok := os.LookupEnv("KO_DOCKER_REPO")
-	var output []byte
-	if ok {
-		re := regexp.MustCompile(DEFAULT_KO_DOCKER_REPO)
-		output = re.ReplaceAll(input, []byte(val))
-	} else {
+	if !ok {
 		return nil, errors.New("KO_DOCKER_REPO is not set")
 	}
+	output := defaultKoDockerRepoRE.ReplaceAll(input, []byte(val))
+
+	// Strip any "namespace: default"s, all resources will be created in
+	// the test namespace using `ko create -n`
+	output = defaultNamespaceRE.ReplaceAll(output, []byte{})
```

Comment:
re that typo above, is this actually doing anything?

Reply:
It wasn't! 🤦‍♂️ 

It apparently needed to actually replace `namespace: default` with `namespace: [the-namespace]`, which it did before, and now does again!